packages/react-sdk/src/components/BoardBar/ExportWhiteboardDialog.tsx

```diff
@@ -54,6 +54,7 @@ export function ExportWhiteboardDialog({
     <Dialog
       aria-labelledby={dialogTitleId}
       aria-describedby={dialogDescriptionId}
+      container={document.getElementById('widget-root')}
```

Comment:
If we create a new dialog we will have forgotten that…

What we can do is create a new Dialog Component in NeoBoard, SDK that has a default value for the container. Then use this Dialog instead. We can also add an [ESLint rule](https://github.com/nordeck/matrix-neoboard/blob/1d3a38ab604584369389e74cd8da0fd9c5c0a913/eslint.config.mjs#L69) to prevent people from using the plain MUI dialog.

Reply:
It should be in toolkit imho just like the rest of this https://github.com/nordeck/matrix-widget-toolkit/blob/main/packages/mui/src/components/MuiThemeProvider/theme.ts#L246-L251

Reply:
So with `document.fullscreenElement ?? document.documentElement` this works without using some IDs. Tested and works. I would say we should then just set the default prop in the toolkit as MTRNord mentioned.

If you want we can have a call about that.

Sorry for the extra round Ahmad :see_no_evil: